src/utils.js

```diff
@@ -324,6 +324,70 @@ go.utils = {
     },
 
 
+// STAGE-BASE HELPERS
+
+    read_subscription: function(im, subscription_id) {
```

Comment:
I prefer consistency myself. Then if you want to standardise a new way move them all at once, instead of mixing.